app/templates/direct-award/view-project.html

```diff
@@ -170,23 +170,10 @@
             ] if project else [])
           },
           {
-            "body": "Complete the Customer Benefits Record form",
+            "body": “Submit a Customer Benefits Record",
             "custom_body_list": [
               {"type": "text", "text":
-                ''.join(["You must fill in the <a href=\"", framework_urls.customer_benefits_record_form_url|e, "\"
-                                                 target=\"_blank\" rel=\"external noopener noreferrer\"
-                                                 data-analytics=\"trackEvent\"
-                                                 data-analytics-category=\"Direct Award\"
-                                                 data-analytics-action=\"External Link\"
-                                                >Customer Benefits Record form</a>
```

Comment:
The existing link has analytics to record an event on GA every time it is clicked; do we know if we will want something similar for the new link?

Reply:
I'm going to go ahead and assume that we want to keep analytics, will update this PR with another commit